src/Avalonia.Input/Avalonia.Input.csproj

```diff
@@ -37,5 +37,8 @@
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.ValueTuple" Version="4.3.1" />
```

Comment:
It's  probably better to move it to `props` file. And make sure that it's listed in nuget dependency list